src/io/image_iter_common.h

```diff
@@ -85,6 +85,14 @@ class ImageLabelMap {
     CHECK(it != idx2label_.end()) << "fail to find imagelabel for id " << imid;
     return mshadow::Tensor<cpu, 1>(it->second, mshadow::Shape1(label_width));
   }
+  /*! \brief find a label for corresponding index, return vector as copy */
+  inline std::vector<float> FindCopy(size_t imid) const {
```

Comment:
FindLabel

Reply:
FindCopy is borrowed from the similar existing code [1], so I use FindCopy for consistency.
But if you do think FindLabel is better, I could update the name, and may open another PR to update the existing FindCopy code.

please tell me if I need to do this.

[1] https://github.com/apache/incubator-mxnet/blob/0fe04e9d778ac9c55e933ff3f21c4ddf28a4a101/src/io/iter_image_det_recordio.cc#L116

Reply:
@piiswrong  hi sincerly, any further suggestions ?